diff -Naur sphinx-2.0.4-release/sphinx.conf.in sphinx-2.0.4-release.oden/sphinx.conf.in
--- sphinx-2.0.4-release/sphinx.conf.in	2011-04-18 16:39:40.000000000 +0200
+++ sphinx-2.0.4-release.oden/sphinx.conf.in	2012-04-03 16:21:33.271286373 +0200
@@ -270,7 +270,7 @@
 	# shell command to invoke xmlpipe stream producer
 	# mandatory
 	#
-	# xmlpipe_command		= cat @CONFDIR@/test.xml
+	# xmlpipe_command		= cat /var/lib/sphinx/test.xml
 
 	# xmlpipe2 field declaration
 	# multi-value, optional, default is empty
@@ -328,7 +328,7 @@
 
 	# index files path and file name, without extension
 	# mandatory, path must be writable, extensions will be auto-appended
-	path			= @CONFDIR@/data/test1
+	path			= /var/lib/sphinx/test1
 
 	# document attribute values (docinfo) storage mode
 	# optional, default is 'extern'
@@ -363,13 +363,13 @@
 	# optional, default is empty
 	# contents are plain text, charset_table and stemming are both applied
 	#
-	# stopwords		= @CONFDIR@/data/stopwords.txt
+	# stopwords		= /var/lib/sphinx/stopwords.txt
 
 
 	# wordforms file, in "mapfrom > mapto" plain text format
 	# optional, default is empty
 	#
-	# wordforms		= @CONFDIR@/data/wordforms.txt
+	# wordforms		= /var/lib/sphinx/wordforms.txt
 
 
 	# tokenizing exceptions file
@@ -378,7 +378,7 @@
 	# plain text, case sensitive, space insensitive in map-from part
 	# one "Map Several Words => ToASingleOne" entry per line
 	#
-	# exceptions		= @CONFDIR@/data/exceptions.txt
+	# exceptions		= /var/lib/sphinx/exceptions.txt
 
 
 	# minimum indexed word length
@@ -577,11 +577,10 @@
 # and may then be overridden in this index definition
 index test1stemmed : test1
 {
-	path			= @CONFDIR@/data/test1stemmed
+	path			= /var/lib/sphinx/test1stemmed
 	morphology		= stem_en
 }
 
-
 # distributed index example
 #
 # this is a virtual index which can NOT be directly indexed,
@@ -631,7 +630,7 @@
 
 	# index files path and file name, without extension
 	# mandatory, path must be writable, extensions will be auto-appended
-	path			= @CONFDIR@/data/rt
+	path			= /var/lib/sphinx/rt
 
 	# RAM chunk size limit
 	# RT index will keep at most this much data in RAM, then flush to disk
@@ -716,16 +715,16 @@
 	# listen			= 192.168.0.1:9312
 	# listen			= 9312
 	# listen			= /var/run/searchd.sock
-	listen			= 9312
+	listen			= 127.0.0.1:9312
 	listen			= 9306:mysql41
 
 	# log file, searchd run info is logged here
 	# optional, default is 'searchd.log'
-	log			= @CONFDIR@/log/searchd.log
+	log			= /var/log/sphinx/sphinx-searchd.log
 
 	# query log file, all search queries are logged here
 	# optional, default is empty (do not log queries)
-	query_log		= @CONFDIR@/log/query.log
+	query_log		= /var/log/sphinx/sphinx-query.log
 
 	# client read timeout, seconds
 	# optional, default is 5
@@ -741,7 +740,7 @@
 
 	# PID file, searchd process ID file name
 	# mandatory
-	pid_file		= @CONFDIR@/log/searchd.pid
+	pid_file		= /var/run/sphinx/sphinx-searchd.pid
 
 	# max amount of matches the daemon ever keeps in RAM, per-index
 	# WARNING, THERE'S ALSO PER-QUERY LIMIT, SEE SetLimits() API CALL
@@ -787,7 +786,7 @@
 	# searchd will (try to) log crashed query to 'crash_log_path.PID' file
 	# optional, default is empty (do not create crash logs)
 	#
-	# crash_log_path		= @CONFDIR@/log/crash
+	# crash_log_path		= /var/log/sphinx/crash
 
 
 	# max allowed per-query filter count
@@ -852,7 +851,7 @@
 	# optional, default is build-time configured data directory
 	#
 	# binlog_path		= # disable logging
-	# binlog_path		= @CONFDIR@/data # binlog.001 etc will be created there
+	# binlog_path		=  /var/lib/sphinx # binlog.001 etc will be created there
 
 
 	# binlog flush/sync mode
diff -Naur sphinx-2.0.4-release/sphinx-min.conf.in sphinx-2.0.4-release.oden/sphinx-min.conf.in
--- sphinx-2.0.4-release/sphinx-min.conf.in	2012-02-28 15:55:53.000000000 +0100
+++ sphinx-2.0.4-release.oden/sphinx-min.conf.in	2012-04-03 16:21:48.580285642 +0200
@@ -26,7 +26,7 @@
 index test1
 {
 	source			= src1
-	path			= @CONFDIR@/data/test1
+	path			= /var/lib/sphinx/test1
 	docinfo			= extern
 	charset_type		= sbcs
 }
@@ -37,7 +37,7 @@
 	type			= rt
 	rt_mem_limit		= 32M
 
-	path			= @CONFDIR@/data/testrt
+	path			= /var/lib/sphinx/testrt
 	charset_type		= utf-8
 
 	rt_field		= title
@@ -54,17 +54,17 @@
 
 searchd
 {
-	listen			= 9312
+	listen			= 127.0.0.1:9312
 	listen			= 9306:mysql41
-	log			= @CONFDIR@/log/searchd.log
-	query_log		= @CONFDIR@/log/query.log
+	log			= /var/log/sphinx/sphinx-searchd.log
+	query_log		= /var/log/sphinx/sphinx-query.log
 	read_timeout		= 5
 	max_children		= 30
-	pid_file		= @CONFDIR@/log/searchd.pid
+	pid_file		= /var/run/sphinx/sphinx-searchd.pid
 	max_matches		= 1000
 	seamless_rotate		= 1
 	preopen_indexes		= 1
 	unlink_old		= 1
 	workers			= threads # for RT to work
-	binlog_path		= @CONFDIR@/data
+	binlog_path		= /var/lib/sphinx
 }
